assets/scss/_layout.scss

```diff
@@ -1,61 +1,5 @@
-html,
 body {
-    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
```

Comment:
Why remove these?

Reply:
Because that's not the font. we are using this.
`$font-family-base:              'Source Sans Pro', sans-serif;`

Reply:
That's only for the headings though.. We do need this for body text. And we also need the others, including the media queries.. I'll ninja-edit them back in on merge! :-)